### ISSUE TITLE
Write an introduction section for the project

### ISSUE THREAD

**Issue by schuylermartin45** [conda-incubator/conda-recipe-manager#187]
We need to add a better introduction section to our `README.md` and self-hosted docs page.

We should answer: _why does this project exist?_ and _how can it help me?_

Here is some text we could adapt.
```
In order for us to improve our package building efforts, we need to automate our process. We have had many engineers (internally and in the community) attempt to automate parts of our process in the past. From those accounts and seeing some of those projects, I believe the core issue is our inability to semantically understand our own recipe format. The Conda Recipe Manager project addresses that issue by being the first tool in our community that can parse recipe files. That has enabled the development of a high-level API for retrieving and editing information in a recipe file.

With this library/API, we can rapidly accelerate the development of other automated tools. We no longer have to re-invent the wheel for every recipe format edge case. No longer do we have to write a series of regular expressions to read very specific portions of the file. If it parses, we can retrieve data from the and edit the file.

Simply put, this project enables others to do complex operations easily.
```
